docs/graph-algorithms/breadth-first-search.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Breadth First Search
+slug: /graph-algorithms/breadth-first-search
+---
+
+## Introduction
+Breadth First Search Algorithm (BFS) is a Popular algorith to find shortest path in unwighted graph
+
+## Explanation
+
+- Given an undirected & Connected Graph (V, E)
+  V: Vertices E: Edges
+- Goal: methodically explore every vertex and every edge
+- “Explore” a graph, turning it into a tree
+- One vertex at a time
+- Expand frontier of explored vertices across its breadth
+- Discovers all vertices at distance k from s before those at distance k+1
+- A level-by-level order traversal
```

Comment:
It doesn't explain anything as well. Prefer using diagrams and representations to explain the same.

Reply:
@HarshCasper  Can I add images in the doc?